### DISCUSSION TITLE
How to I use multiple Pocketbase server on My VPS machine.

### DISCUSSION THREAD

**Discussion by manishonc** [pocketbase/pocketbase#1367] [Q&A] [answered]
How to I use multiple Pocketbase server on My VPS machine.

Like One .. apione.domain.com
apitwo.domain.com

**Comment by ganigeorgiev** [comment, marked as answer]
You can start multiple PocketBase instances from the same executable but with different `pb_data` dir and address like this:

```sh
./pocketbase serve --dir="/path/to/pb_data1" --http="127.0.0.1:8091" & \
./pocketbase serve --dir="/path/to/pb_data2" --http="127.0.0.1:8092" &

# to kill all instances
sudo killall pocketbase
```

But if you want to bind to multiple domains each on the same 80 and 443 port you'll need to setup a reverse-proxy, eg. nginx (_`SO_REUSEPORT/SO_REUSEADDR` could also be an option but I've never used it_).

For a sample nginx proxy config you can check - https://pocketbase.io/docs/going-to-production/#using-reverse-proxy

**Reply by manishonc** [reply]
Thankyou
